security/firely-auth/firely-auth-sso.rst

```diff
@@ -87,6 +92,8 @@ Configuring a new client application in Azure Active Directory (Azure AD) using
     - Select "Endpoints"
     - One of the displayed OAuth 2.0 endpoints can be used as the authority in the settings. It should look like this: ``https://login.microsoftonline.com/<Directory (tenant) ID of the registered application>/v2.0``.
 
+#. Optional: Expose the `groups <https://learn.microsoft.com/en-us/entra/identity-platform/optional-claims?tabs=appui#configure-groups-optional-claims>`_ in the ID token if the SSO auto-provisioning is restricted to certain security groups.
```

Comment:
Are there any considerations for the different types of group claims that can be exposed? There are for instance several options for the ID, Acces, and SAML types.